test/cli/main_test.rb

```diff
@@ -2,12 +2,17 @@
 
 class CliMainTest < CliTestCase
   test "setup" do
+    invoke_options = { "config_file" => "test/fixtures/deploy_simple.yml", "version" => "999", "skip_hooks" => false }
+
```

Comment:
That seems unneeded as well:

```suggestion
```

Reply:
Yes, but at the same time I don't see tests, which cover `skip_push` option for the `setup` task.

Reply:
Yes, sure, my bad. Removed